torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -879,16 +879,116 @@ at::Tensor& XLANativeFunctions::arange_out(const at::Scalar& start,
   return out;
 }
 
+at::Tensor as_strided_eliminate_one_dim_fast_path(
```

Comment:
We can make this function `static` to hide it from the symbols exposed by the library. It's generally a good practice to make all C++ functions `static` or put them in an anonymous namespace unless we want it to be used by other `.cpp` files.